WarmUp1/frontBack/src/index.ts

```diff
@@ -0,0 +1,23 @@
+/*
+Given a string, return a new string where the first and last chars have been exchanged.
+
+
+frontBack("code") → "eodc"
+frontBack("a") → "a"
+frontBack("ab") → "ba"
+*/
+
+export function frontBack(strParameter: string): string {
```

Comment:
Naming of parameters are off again, we know its a parameter based on where it is declared